src/components/tree/phyloTree/grid.js

```diff
@@ -334,17 +342,20 @@ export const addGrid = function addGrid() {
   timerEnd("addGrid");
 };
 
-
-export const removeTemporalSlice = function removeTemporalSlice() {
```

Comment:
This removed function is still set as a prototype in `phyloTree.js` and called by the function `modifySVGInStages`. This will result in a complete app crash when (e.g.) the tree layout it changed

Reply:
Oops, I didn't realize this was used on the outside, I should have checked given that it was `export`ed. I'll bring this back.